### PR TITLE
global: add nharraud as maintainer

### PR DESCRIPTION
**Pull request by nharraud**
Signed-off-by: Nicolas Harraudeau nicolas.harraudeau@cern.ch

### CODE REVIEW

File: .lgtm [hunk 1]
```diff
@@ -0,0 +1,3 @@
+approvals = 1
+pattern = "(?i)LGTM"
+self_approval_off = true
```
Comment (jirikuncar) [inline .lgtm]: since you are the only maintainer you might want to enable self approvals

File: MANIFEST.in [hunk 2]
```diff
@@ -32,7 +32,9 @@ include *.rst
 include *.sh
 include *.txt
 include docs/requirements.txt
+include .lgtm
```
Comment (jirikuncar) [inline MANIFEST.in:31]: Can you also resort the `MANIFEST.in` and remove old comments?

**Comment by jirikuncar** [comment]
@nharraud please follow [commit message guidelines](http://invenio.readthedocs.io/en/latest/technology/git.html#r2-remarks-on-commit-log-messages). You can use commit message from the example you have received via email: `docs: MAINTAINERS file and LGTM configuration` (see https://github.com/inveniosoftware/invenio-github/pull/35)

**Comment by nharraud** [comment]
@jirikuncar done, thanks for the review :)

**Pull request merged**